<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Constants_Dialog</class>
 <widget class="QDialog" name="Constants_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Constants</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close_button">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>217</width>
     <height>169</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lodging_tax_edit">
      <property name="text">
       <string>2.50</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="aaa_discount_label">
      <property name="text">
       <string>AAA Discount</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="sales_tax_edit">
      <property name="text">
       <string>15.25</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lodging_tax_label">
      <property name="text">
       <string>Lodging Tax</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="sales_tax_label">
      <property name="text">
       <string>Sales Tax</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="aaa_discount_edit">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbms_discount_label">
      <property name="text">
       <string>Manager's Special</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="soe_commission_edit">
      <property name="text">
       <string>17</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="expedia_discount_label">
      <property name="text">
       <string>Expedia Discount</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="expedia_commission_label">
      <property name="text">
       <string>Expedia Commission</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="soe_discount_edit">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lbms_discount_edit">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="percent_label1">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="percent_label2">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Flat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="percent_label3">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="percent_label4">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="percent_label5">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="percent_label6">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sales_tax_edit</tabstop>
  <tabstop>lodging_tax_edit</tabstop>
  <tabstop>aaa_discount_edit</tabstop>
  <tabstop>lbms_discount_edit</tabstop>
  <tabstop>soe_discount_edit</tabstop>
  <tabstop>soe_commission_edit</tabstop>
  <tabstop>save_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
